src/utils/helpers.js

```diff
@@ -218,8 +218,25 @@ const helpers = {
       return '';
     }
     return parts[parts.length - 1];
-  }
+  },
 
+  /**
+   * Render value to integer or decimal
+   *
+   * @param {number} amount Amount to render
+   * @param {boolean} isInteger If it's an integer or decimal
+   *
+   * @return {string} rendered value
+   * @memberof Helpers
+   * @inner
+   */
+  renderValue(amount, isInteger) {
```

Comment:
I think this method should be `renderValue(amount, decimals)`. But I understand that it just calls the library which does not support decimals yet, right?

Reply:
Yes, exactly. Right now the lib supports only 2 decimals or 0 decimals. I guess if we add custom decimal places we will change that (and many other places). I think we should leave that for the future, what do you think?

Reply:
I agree.